test/integration/rules/aria-empty-table-header/aria-empty-table-header.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>empty-table-header test</title>
+		<meta charset="utf8" />
+		<script src="/axe.js"></script>
+	</head>
+
+	<body>
+		<th id="pass1">Ok</th>
```

Comment:
This won't work and is probably why the tests are failing. Table elements (td, th, etc.) cannot be created outside of a table and the browser will just ignore them and turn them into text:

```
<text>Ok
  rowheader with name
  columnheader with name
  
  
    </text>
```

Need to wrap these elements inside a valid table.